tests/__snapshots__/test_help_messages.ambr

```diff
@@ -914,487 +914,6 @@
-  │                                           authentication (MFA)               │
-  │ --enable-diag                             Run python connector diagnostic    │
-  │                                           test                               │
-  │ --diag-log-path                     TEXT  Diagnostic report path             │
-  │ --diag-allowlist-path               TEXT  Diagnostic report path to optional │
-  │                                           allowlist                          │
-  ╰──────────────────────────────────────────────────────────────────────────────╯
-  ╭─ Global configuration ───────────────────────────────────────────────────────╮
-  │ --format           [TABLE|JSON]  Specifies the output format.                │
-  │                                  [default: TABLE]                            │
-  │ --verbose  -v                    Displays log entries for log levels `info`  │
-  │                                  and higher.                                 │
-  │ --debug                          Displays log entries for log levels `debug` │
-  │                                  and higher; debug logs contains additional  │
-  │                                  information.                                │
-  │ --silent                         Turns off intermediate output to console.   │
-  ╰──────────────────────────────────────────────────────────────────────────────╯
-  
-  
-  '''
-# ---
-# name: test_help_messages[git]
-  '''
-                                                                                  
-   Usage: default git [OPTIONS] COMMAND [ARGS]...                                 
-                                                                                  
-   Manages git repositories in Snowflake.                                         
-                                                                                  
-  ╭─ Options ────────────────────────────────────────────────────────────────────╮
-  │ --help  -h        Show this message and exit.                                │
-  ╰──────────────────────────────────────────────────────────────────────────────╯
-  ╭─ Commands ───────────────────────────────────────────────────────────────────╮
-  │ copy           Copies all files from given state of repository to local      │
-  │                directory or stage.                                           │
-  │ fetch          Fetch changes from origin to snowflake repository.            │
-  │ list-branches  List all branches in the repository.                          │
-  │ list-files     List files from given state of git repository.                │
-  │ list-tags      List all tags in the repository.                              │
-  │ setup          Sets up a git repository object.                              │
-  ╰──────────────────────────────────────────────────────────────────────────────╯
-  
-  
```

Comment:
Ran with `--snapshot-update` 🤷

Reply:
That's weird, git commands should be visible (they're turned on in default config.toml) 🤔

Reply:
It's not related to this PR, I've created a jira ticket for that